CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+* **2015-08-27** **BC break** Missing parameter in `DocumentManagerInterface::getDocumentsByPhpcrQuery`
+                 added. If you have your own implementation that is not based
+                 on DocumentManagerDecorator, you will need to adjust that method.
```

Comment:
a typical bugfix BC break.

i think this is really unlikely to hit people, as they probably where using the decorator and then unless they overwrite the getDocumentsByPhpcrQuery method will not notice the change.